lib/api/request/kuzzleRequest.ts

```diff
@@ -604,6 +606,50 @@ export class KuzzleRequest {
     return this._getObject(this.input.args, name, name, def);
   }
 
+  /**
+   * Gets a parameter from a request arguments and check if the date is an ISO8601 format date or is valid regarding
+   * a given custom format (example : YYYY-MM-DD).
+   *
+   * @param name parameter name.
+   * @param format optional parameter to check if the date is valid regarding a format. If not set, the format checked
+   * is ISO8601.
+   * @throws {api.assert.missing_argument} If parameter not found and no default
+   *                                       value provided
+   * @throws {api.assert.invalid_type} If parameter value is not a valid date.
+   */
+  getDate (name: string, format?: string): string {
+    const args = this.input.args;
+    if (args[name] === undefined) {
+      throw assertionError.get('missing_argument', name);
+    }
+    if (format && moment(args[name], format, true).isValid() === false) {
+      throw assertionError.get('invalid_type', name, 'date');
+    }
+    if ( moment(args[name], moment.ISO_8601).isValid() === false) {
```

Comment:
Nitpicking: 
 - usually we don't add blank space after parenthesis
 - usually we use the `!` modifier instead of `=== false`
 
```suggestion
    if (! moment(args[name], moment.ISO_8601).isValid()) {
```